### ISSUE TITLE
[Question] About weight decay in MOPO implementation

### ISSUE THREAD

**Issue by linhlpv** [yihaosun1124/OfflineRL-Kit#13]
Hi @yihaosun1124, 
Thank you for your amazing works.

I'm reproducing MOPO results in D4RL tasks using your code. I see that you use the weight decay and L2 regularization for learning the ensemble transition functions. I just wonder about the reason for this implementation, and could we remove the decay loss (l2 regularization) or it's mandatory? Could you please help me about this question?

Again, thank you so much for your repository.
Best,
Linh

**Comment by yihaosun1124** [comment]
Hi! Sorry for my late reply!
The setting of the dynamics model follows MOPO's official implementation (https://github.com/tianheyu927/mopo/blob/master/mopo/models/constructor.py). I think you may try to remove it but it may influence other hyper-parameters, like "rollout length" and "penalty coef" since original hyper-parameters may not be suitable for this new model any more.
